flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/config/AutoScalerOptions.java

```diff
@@ -153,4 +154,12 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                     .defaultValue(MetricAggregator.MAX)
                     .withDescription(
                             "Metric aggregator to use for busyTime metrics. This affects how true processing/output rate will be computed. Using max allows us to handle jobs with data skew more robustly, while avg may provide better stability when we know that the load distribution is even.");
+
+    public static final ConfigOption<List<String>> VERTEX_EXCLUDE_IDS =
+            autoScalerConfig("vertex.exclude.ids")
+                    .stringType()
+                    .asList()
+                    .defaultValues(new String[0])
+                    .withDescription(
+                            "A (semicolon-separated) list of vertex ids in hexstring to disable for autoscaler to ignore for scaling.");
```

Comment:
```suggestion
                            "A (semicolon-separated) list of vertex ids in hexstring for which to disable scaling. Caution: For non-sink vertices this will still scale their downstream operators until https://issues.apache.org/jira/browse/FLINK-31215 is implemented.");
```